features/createroom/impl/src/main/kotlin/io/element/android/features/createroom/impl/root/CreateRoomRootNode.kt

```diff
@@ -72,8 +70,8 @@ class CreateRoomRootNode @AssistedInject constructor(
             state = state,
             modifier = modifier,
             onClosePressed = this::navigateUp,
-            onNewRoomClicked = callback::onCreateNewRoom,
-            onOpenDM = callback::onStartChatSuccess,
+            onNewRoomClicked = ::onCreateNewRoom,
+            onOpenDM = ::onStartChatSuccess,
```

Comment:
The compose compiler (or Kotlin compiler?) did not like the previous version

Reply:
We could just use `= { callback.onCreateNewRoom() }` there, but this is fine too.

Reply:
Yes, but I prefer to keep the same pattern everywhere.